bot/exts/moderation/voice_gate.py

```diff
@@ -28,12 +29,26 @@
     "activity_blocks": f"have been active for fewer than {GateConf.minimum_activity_blocks} ten-minute blocks",
 }
 
+VOICE_PING = (
+    "Hello, {0}! Wondering why you can't talk in the voice channels? "
+    "Use the `!voiceverify` command in here to verify. "
+    "If you don't yet qualify, you'll be told why!"
+)
+
 
 class VoiceGate(Cog):
     """Voice channels verification management."""
 
-    def __init__(self, bot: Bot):
+    # RedisCache[t.Union[discord.User.id, discord.Member.id], t.Union[discord.Message.id, bool]]
+    redis_cache = RedisCache()
```

Comment:
I feel like there should be a comment here explaining what the values represent. There are some bits of information scattered around the cog, but better to have them here.

Reply:
Added in latest commit